### ISSUE TITLE
Wrong Hexahedron element visualisation 

### ISSUE THREAD

**Issue by Sidaty1** [mimesis-inria/caribou#117]
When we test the forcefield using the Sofa GUI, we observe a wrong draw of hexahedron elements, this could be related to the topology change between Fenics and Caribou elements

**Comment by Ziemnono** [comment]
Hi @Sidaty1 and @jnbrunet,
I changed back the definition of the element to its original one. It will allow merging without breaking the existing Caribou implementation of Hexahedron.
Actually, if you [change the definition of the Hexahedron](https://github.com/jnbrunet/caribou/blob/b30b401d03890e2e7dbfb79c96d9bce2a6875a49/src/Caribou/Geometry/Hexahedron.h#L175) you can have a good visualization of the FEniCS hexahedron.

We have 2 choices:
- Either we create a FEniCS hexahedron element (actually the element is just used for defining the geometry as it does not need to create shapes functions or gauss points, already managed by FEniCS). But we may have to template it for the ```CaribouTopology``` and ```CaribouForceField``` (a lot of copy/paste just for visualization).
- Or we change the definition of the [function get_boundary_elements_nodes()](https://github.com/jnbrunet/caribou/blob/b30b401d03890e2e7dbfb79c96d9bce2a6875a49/src/Caribou/Geometry/Hexahedron.h#L167), to allow some indices swapping for correcting the face orientation.

I can take care of the first option easily but I do not feel confident about the second one...
Please let me both know what you prefer, have a nice evening.

**Comment by jnbrunet** [comment]
I would go for the first option. I think we should first merge the master branch in the fenics one as I just merged the element order refactoring PR](https://github.com/jnbrunet/caribou/pull/108). I can give it a try this evening.

Unfortunately, while not too difficult, we indeed have to specialize `CaribouTopology` and `CaribouForceField` for every new element type. I have some plan to remove the need of CaribouForceField specializations at some point and replace it by virtual calls.
